packages/authorization-updater/src/utils.ts

```diff
@@ -103,3 +120,54 @@ export const getPurposeVersionFromEvent = (
 
   return { purposeId: purpose.id, purposeVersion };
 };
+
+export const getClientFromEvent = (
+  msg: {
+    data: {
+      client?: ClientV2;
+    };
+  },
+  eventType: string
+): Client => {
+  if (!msg.data.client) {
+    throw missingKafkaMessageDataError("client", eventType);
+  }
+
+  return fromClientV2(msg.data.client);
+};
+
+export const apiClientKind = {
+  consumer: "CONSUMER",
+  api: "API",
+} as const;
+export const ApiClientKind = z.enum([
+  Object.values(apiClientKind)[0],
+  ...Object.values(apiClientKind).slice(1),
+]);
+export type ApiClientKind = z.infer<typeof ApiClientKind>;
```

Comment:
Ups, forgot to tell you to also move this in models taking it from the generated api